src/typechecker/FStar.TypeChecker.DeferredImplicits.fs

```diff
@@ -138,7 +138,7 @@ let solve_deferred_to_tactic_goals env g =
         in
         let goal_ty = U.mk_eq2 (env.universe_of env_lax t_eq) t_eq tp.lhs tp.rhs in
         let goal, ctx_uvar, _ =
-            Env.new_implicit_var_aux reason tp.lhs.pos env goal_ty Strict None
+            Env.new_implicit_var_aux reason tp.lhs.pos env goal_ty Allow_untyped None
```

Comment:
Can you say a bit more about this change?

Reply:
I can't remember when this change was first introduce... But it's been on the steel branch for a long time.
I think it's what allows us to avoid retyping layered effects implicits in phase 2

Reply:
ah, I misread this diff ... I though it was a diff relative to steel.

Reply:
May be we should have another case `Tactic_implicit` or something, because it is not untyped, just that the well-typedness of the solution is enforced by the tactic.

Reply:
An additional case would be more readable, although would it semantically change something here? This goal is already deferred to tactic

Reply:
Nothing semantic, you can merge the PR as is, I can do this change later sometime.